app/scripts/lib/setupSentry.js

```diff
@@ -506,53 +506,52 @@ export default function setupSentry({ release, getState }) {
   });
 
   /**
-   * As long as a reference to the Sentry Hub can be found, and the user has
-   * opted into MetaMetrics, change the autoSessionTracking option and start
-   * a new sentry session.
+   * As long as a reference to the Sentry Hub can be found, and the session has
+   * not been started, start a new sentry session.
    */
   const startSession = async () => {
-    const hub = Sentry.getCurrentHub?.();
-    const options = hub.getClient?.().getOptions?.() ?? {};
-    if (hub && (await getMetaMetricsEnabled()) === true) {
```

Comment:
It looks like the metrics enabled check has been removed from all three of these. Was that intentional?

Reply:
I added that check back in. Confusingly, the state switches between `toggleSession` and `startSession` or `endSession`, that's why I'd initially remove them. However, we do need that check for onboarding to work correctly.